models/uniswap/ethereum/uniswap_v1_ethereum_trades.sql

```diff
@@ -5,13 +5,17 @@
     materialized = 'incremental',
     file_format = 'delta',
     incremental_strategy = 'merge',
-    unique_key = ['block_date', 'unique_trade_id'],
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
     post_hook='{{ expose_spells(\'["ethereum"]\',
                                 "project",
                                 "uniswap_v1",
                                 \'["jeff-dude"]\') }}'
     )
 }}
+
+{% set project_start_date = '2018-11-01' %}
+{% set weth_address = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' %}
```

Comment:
@masquot and i were thinking we could even use a few (like ```weth_address```) as global variables in the dbt_project yaml file. keeps it even more simple. just need to ensure it's intuitive for users to find and know it exists

Reply:
but only truly global values, of course, to avoid that file getting overloaded (which was a prior concern if putting contract specific variables there)

Reply:
sounds like a good idea !